Gemfile

```diff
@@ -79,4 +79,5 @@ group :development, :test do
 end
 
 gem 'rails-controller-testing'
-gem 'rspec-support', '3.12.1'
+# gem 'rspec-support', '3.12.6'
+gem 'rspec-mocks', '3.12.6'
```

Comment:
- I don't see Devise as one of the gems. Please install it

Reply:
Hi @afanimartin , I clumsily submitted this pull request on day 2 instead of submitting it on day 1. Unfortunately, there was no way to undo this action. Can we cancel this review?